### PR TITLE
Removed tabs, made style changes to article cards

### PR DESCRIPTION
**Pull request by isis-meng**
Link to Trello card for [description section](https://trello.com/c/2iTrxhj5/62-rivalry-edition-description-section)
Link to Trello card for [articles](https://trello.com/c/uqdtG3eF/59-rivalry-edition-articles)
Misc: Minor style changes to landing page

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Main from "./components/Main";
 import Article from "./components/Article/Article";
 import Landing_Page from "./components/Landing_Page/Landing_Page";
-import Tabs from "./components/Tabs";
+// import Tabs from "./components/Tabs";
```
Comment (parthd4) [inline src/js/index.js:7]: nit: can you delete this?

File: src/js/components/ArticleGrid.jsx [hunk 2]
```diff
@@ -25,110 +25,39 @@ const ArticleGrid = ({ tabType }) => {
...
-    }
+    return (
+      <>
+        <p>
+          For the second year, The Michigan Daily and The Ohio State Lantern are
+          producing a joint publication with stories about our schools' historic
+          football rivalry. We're launching this work alongside a competitive
+          fundraiser to help us cover our operating costs. Whether you're a
+          Buckeye or a Wolverine, please consider donating to support
+          award-winning student journalism.
```
Comment (parthd4) [inline src/js/components/ArticleGrid.jsx:39]: Can you set this up to get the paragraph from the [google doc](https://docs.google.com/document/d/1ZHkeFQJzjH_vm3GTB9ZSRGS_lZQHcmlkI2Qf5Rp6HSM/edit)? I would create a new section under the homepage section that has `description: For the second year....`

File: src/js/components/ArticleGrid.jsx [hunk 3]
```diff
@@ -25,110 +25,39 @@ const ArticleGrid = ({ tabType }) => {
...
   const getData = () => {
-    if (tabType === "All") {
-      return getAllData();
-    } else if (tabType === "UM") {
-      return data.michigan.articles;
-    } else {
-      return data.ohio.articles;
-    }
+    return getAllData();
   };
```
Comment (parthd4) [inline src/js/components/ArticleGrid.jsx:29]: Can we delete the `getData` function to simplify the code? Right now `getData` just calls `getAllData` and does nothing else. And then on line 66 call `getAllData` directly.

**Review by parthd4 — APPROVED** [review]
Just a couple minor changes and then you're good to go! Nice work 🔥

**Review by parthd4 — APPROVED** [review]

File: src/js/components/ArticleGrid.jsx [hunk 4]
```diff
@@ -24,117 +24,36 @@ const ArticleGrid = ({ tabType }) => {
...
-          <br />
-          <a href="https://buckeyefunder.osu.edu/project/21918" target="_blank">
-            <img src={lantern}></img>
-          </a>
-        </>
-      );
-    }
+    return (
+      <>
+        <div className="intro-desc">{data.homepage.description}</div>
```
Comment (rimazmk) [inline src/js/components/ArticleGrid.jsx:30]: Can you make the text for this left aligned + a bit larger? Also could you increase the spacing between this an d the video?

**Review by rimazmk — APPROVED** [review]

**Comment by rimazmk** [comment]
Great job! Just made a minor suggestion you can decide to accept/ignore

File: src/js/components/ArticleGrid.jsx (hunk 4 above)
Comment (isis-meng) [reply src/js/components/ArticleGrid.jsx:30]: Pushed changes, let me know what you think :)

Comment (rimazmk) [reply src/js/components/ArticleGrid.jsx:30]: looks great!

**Pull request merged**